src/docs/_sass/_docs-block.scss

```diff
@@ -8,43 +9,69 @@
     font-weight: normal;
     margin: 0;
   }
+
   &__info {
     flex: 0 1 auto;
     box-sizing: border-box;
     padding: 0 20px 0 0;
     width: 230px;
   }
+
   &__content {
     flex: 1 1 auto;
+
+    &--logo-preview {
+      font-size: 150px;
+    }
+
     &--centered {
       display: flex;
       align-items: center
     }
+
     &--align-top {
       & > * {
         vertical-align: top;
       }
     }
+
     &:hover {
+      position: relative;
+
       .docs-highlight-placeholder {
         outline: 2px dashed white;
         box-shadow: black 0 0 10px 0;
       }
+
+      &:before {
```

Comment:
This thing interferes with other mouse events.
We need to figure out a better way of triggering the rhythm guidelines, e.g. global checkbox